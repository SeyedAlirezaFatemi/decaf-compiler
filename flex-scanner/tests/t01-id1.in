abcdefg
Stephen
stephen_hawking
six_dot_035
foo_
covid_19
