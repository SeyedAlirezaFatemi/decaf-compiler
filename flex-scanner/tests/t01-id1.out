T_ID abcdefg
T_ID Stephen
T_ID stephen_hawking
T_ID six_dot_035
T_ID foo_
T_ID covid_19
